integration_tests/test/dark-mode.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { test } from "fixtures";
+import { TEST_PROJECT } from "~client/_test/test.data.helper";
+import { mockGetProjects } from "utils/mocks";
+
+// Run tests in this file with dark mode preferred
+test.use({ colorScheme: "dark" });
+
+const project = TEST_PROJECT;
+
+test.beforeEach(async ({ page }) => {
+  await mockGetProjects(page, [project]);
+});
+
+test("get dark screens", async ({ page, vrt }) => {
```

Comment:
you could use loop to make it separate test like here 
https://github.com/Visual-Regression-Tracker/frontend/blob/master/integration_tests/test/userList.spec.ts#L28

Reply:
I am afraid that at this point it would still be premature optimization...